src/command.ts

```diff
@@ -131,7 +122,7 @@ export default abstract class Command {
   protected static _globalFlags: Interfaces.FlagInput
 
   static get globalFlags(): Interfaces.FlagInput {
-    return this._globalFlags
+    return Object.assign(this._globalFlags ?? {}, this.enableJsonFlag ? {...jsonFlag} : {})
```

Comment:
I placed the eval of enableJsonFlag here so tests would pass. The format tests were failing missing the Global Flags section.